packages/prisma-client/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@webstudio-is/prisma-client",
+  "version": "0.0.1",
+  "description": "Webstudio Prisma layer",
+  "author": "Webstudio <github@webstudio.is>",
+  "homepage": "https://webstudio.is",
+  "scripts": {
```

Comment:
is it supposed to build generate/migrate on postinstall?

Reply:
If we want yes

Reply:
done!